src/plugins/data-card/components/sort-area.tsx

```diff
@@ -9,10 +14,40 @@ interface IProps {
 export const DataCardSortArea: React.FC<IProps> = ({ model }) => {
   const content = model.content as DataCardContentModelType;
   const sortById = content.selectedSortAttributeId;
-  const sortByName = sortById ? content.dataSet.attrFromID(sortById).name : " ";
+
+  const attrsSnap = getSnapshot(content.attributes);
+  const allAttrValues = attrsSnap.filter((a) => a.id === sortById)[0].values;
+  const uniqeOrderedValues = orderBy(uniq(allAttrValues));
+
+   // if one of the categories is a category for no value, put this stack last
+  uniqeOrderedValues.includes("") && uniqeOrderedValues.push(uniqeOrderedValues.shift());
+
+  const renderPlaceholderCells = () => {
+    const rowsNeeded = Math.ceil(uniqeOrderedValues.length / 3);
+    const placeholders = (rowsNeeded * 3) - uniqeOrderedValues.length;
+    return (
+      <>
+        { placeholders > 0 && <SortStackPlaceholder /> }
+        { placeholders === 2 && <SortStackPlaceholder /> }
+      </>
+    );
```

Comment:
Or even more likely, determining the number of columns dynamically based on available width.

Reply:
yeah we aren't doing the dynamic thing right now because I specifically didn't ask for it. Not clear if it's better to add more columns or make the cards/columns wider.

I'm more concerned about the misspelling of 'unique' in the name uniqeOrderedValues...